src/envo/envo-edit.owl

```diff
@@ -39732,6 +39740,62 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_04000018> <http://purl.obolibrar
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_06105103> <http://orcid.org/0000-0003-1072-8081>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_06105103> "2021-04-08T14:01:58Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_06105103> "While the definition is in accordance with the common use of this term (as per GESAMP report # 99), alternative size classifications exist. An alternative classification is a piece of plastic which is 25 - 50 mm in its longest dimension."@en)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105103> "macroplastic particle"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105103> <http://purl.obolibrary.org/obo/ENVO_01000776>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105104> (mesoplastic particle)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://www.gesamp.org/publications/guidelines-for-the-monitoring-and-assessment-of-plastic-litter-in-the-ocean>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105104> "A piece of plastic which is 5 - 25 mm in its longest dimension."@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_06105104> <https://orcid.org/0000-0002-2996-719X>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_06105104> <https://orcid.org/0000-0002-8166-0486>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_06105104> <http://orcid.org/0000-0003-1072-8081>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_06105104> "2021-04-08T14:04:53Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_06105104> "While the definition is in accordance with the common use of this term (as per GESAMP report # 99), alternative size classifications exist. Alternatives classifications are a piece of plastic which is less than 25 mm in its longest dimension, or a piece of plastic which is 1 - 25 mm in its longest dimension."@en)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105104> "mesoplastic particle"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105104> <http://purl.obolibrary.org/obo/ENVO_01000776>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105105> (nanoplastic particle)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://www.gesamp.org/publications/guidelines-for-the-monitoring-and-assessment-of-plastic-litter-in-the-ocean>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105105> "A piece of plastic which has a diameter less than 1 µm."@en)
```

Comment:
Note the comments above - same applies here.
Additionally, why the switch to diameter rather than the longest dimension? The comment suggests that GESAMP assumes a spherical shape in general - does this apply only to this term? 

Also note that there are [generalisations ](https://en.wikipedia.org/wiki/Diameter#Generalizations)of "diamater" beyond circles and spheres.

My recommendation is to maintain the "longest dimension"  or note that we use the generalised form of diameter in all these definitions. The GESAMP assumption seems to be a convenience, which is slightly hard to translate here.